woocommerce/woocommerce
# Add missing styles for the progress header

### All Submissions:






### Changes proposed in this Pull Request:



Closes #34509  .

### How to test the changes in this Pull Request:

1. Check out new branch from trunk (beta version of WooCommerce 7).
2. Navigate to "Customers" screen.
3. Click "Finish Setup" button in header.
4. Observe styled header.


### Other information:

-   [] Have you successfully run tests with your changes locally?



### FOR PR REVIEWER ONLY:
